### ISSUE TITLE
update git software installation instructions, mac

### ISSUE THREAD

**Issue by ozanj** [anyone-can-cook/rclass2#23]
Hey team,
@augias @joycehnguy @xochilthlopez 

want to update instructions for installing software for git, separately for mac and pc. I'll do PC. can one of you do mac? 

this may not require any change from current. see link below for where to update:

https://anyone-can-cook.github.io/rclass2/lectures/git_and_github/git_and_github.html#14_Installation_and_running_shell_commands_from_RStudio

basically, seems to suggest that no new software is needed for mac users. I know this is true for terminal, but surprised that git is already installed.
